src/meson.build

```diff
@@ -134,6 +147,7 @@ if enable_gui
   executable('lxi-gui', lxi_gui_sources,
     dependencies: lxi_gui_deps,
     install: true,
+    win_subsystem: 'windows' # on windows it start wuth cmd without this settings.
```

Comment:
Small type "wuth".

You mean without this the lxi commandline tool is started instead of lxi-gui through whatever windows shortcut is created?

Reply:
No, in windows lxi-gui start also with other cmd prompt window. Default value is console. Thats why I chaneg it

Reply:
?? So when you start lxi-gui through clicking in the Windows gui it will start a cmd prompt window??